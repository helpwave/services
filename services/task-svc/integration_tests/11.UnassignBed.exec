#!/bin/bash

echo "Assigned Patient:"
grpcurl --plaintext -H "X-Organization: 3b25c6f5-4705-4074-9fc6-a50c28eba406" \
	-H "Authorization: Bearer eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm5hbWUiOiJUZXN0aW5lIFRlc3QiLCJuaWNrbmFtZSI6InRlc3RpbmUudGVzdCIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19" \
	-d '{"id": "c1c104f0-52c5-46d3-9441-33d0b0c03f87"}' \
	localhost:3001 \
	proto.services.task_svc.v1.PatientService/UnassignBed

echo "(reassign)"
grpcurl --plaintext -H "X-Organization: 3b25c6f5-4705-4074-9fc6-a50c28eba406" \
	-H "Authorization: Bearer eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm5hbWUiOiJUZXN0aW5lIFRlc3QiLCJuaWNrbmFtZSI6InRlc3RpbmUudGVzdCIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19" \
	-d '{"id": "c1c104f0-52c5-46d3-9441-33d0b0c03f87", "bed_id": "cb92bad3-68b2-49c1-bfac-494ff18b57f3"}' \
	localhost:3001 \
	proto.services.task_svc.v1.PatientService/AssignBed > /dev/null

echo "Unassigned Patient:"
grpcurl --plaintext -H "X-Organization: 3b25c6f5-4705-4074-9fc6-a50c28eba406" \
	-H "Authorization: Bearer eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm5hbWUiOiJUZXN0aW5lIFRlc3QiLCJuaWNrbmFtZSI6InRlc3RpbmUudGVzdCIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19" \
	-d '{"id": "2e830d2f-69c9-4e38-bd4b-ab43decc7ae3"}' \
	localhost:3001 \
	proto.services.task_svc.v1.PatientService/UnassignBed 2>&1 || echo ""

echo "Patient does not exist:"
grpcurl --plaintext -H "X-Organization: 3b25c6f5-4705-4074-9fc6-a50c28eba406" \
	-H "Authorization: Bearer eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm5hbWUiOiJUZXN0aW5lIFRlc3QiLCJuaWNrbmFtZSI6InRlc3RpbmUudGVzdCIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19" \
	-d '{"id": "98fa077f-4bc2-4c54-82fa-57d6728db45c"}' \
	localhost:3001 \
	proto.services.task_svc.v1.PatientService/UnassignBed
