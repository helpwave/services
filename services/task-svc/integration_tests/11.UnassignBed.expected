Assigned Patient:
{}
(reassign)
Unassigned Patient:
{}
Patient does not exist:
{}
